deploy/charts/cert-manager/templates/rbac.yaml

```diff
@@ -10,14 +10,19 @@ metadata:
     heritage: {{ .Release.Service }}
 rules:
   - apiGroups: ["certmanager.k8s.io"]
-    resources: ["certificates", "certificates/finalizers", "issuers", "clusterissuers", "orders", "orders/finalizers", "challenges"]
+    resources: ["certificates", "certificates/finalizers", "issuers", "issuers/finalizers","clusterissuers", "orders", "orders/finalizers", "challenges",  "challenges/finalizers"]
     verbs: ["*"]
   - apiGroups: [""]
-    resources: ["configmaps", "secrets", "events", "services", "pods"]
+    resources: ["configmaps", "secrets", "secrets/finalizers","events", "services", "services/finalizers","pods", "pods/finalizers"]
     verbs: ["*"]
   - apiGroups: ["extensions"]
-    resources: ["ingresses"]
+    resources: ["ingresses", "ingresses/finalizers", "ingresses/status"]
```

Comment:
I'm surprised we need any of these as we don't use finalizers or the status subresource on ingresses 🤔

Reply:
We need the finalzer for this as it fire this error 
```ingress-shim controller: Re-queuing item "hello-world-ssl/prod-ingress" due to error processing: certificates.certmanager.k8s.io "prod-ingress-ssl" is forbidden: cannot set blockOwnerDeletion if an ownerReference refers to a resource you can't set finalizers on: no RBAC policy matched, <nil>```

The status, however, does not seems to be needed